server/models/Profile.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+
+const dayAvalSchema = new mongoose.Schema({
```

Comment:
```suggestion
const dayAvailabilitySchema = new mongoose.Schema({
```
No need to resort to unconventional abbreviations 👍🏼